src/property.h

```diff
@@ -310,8 +310,10 @@
 #define ALO_HIGH 2
 #define ALO_OFF 3
 
-#if defined(CONFIG_5D3)
+// was guarded for 5D3 / 6D, but R series seem to use it too.
```

Comment:
This exposes only property ID. To use it, you need to enable a code path separately by using `GENERIC_GET_HTP` in `cfn.c` in camera platform dir. Due to that I don't see any downsides of doing this. 

Worst case we can do `ifdef 5D3 / D6 / D678`.

Reply:
Yes, I think I'm just being cautious because I don't understand why they were in blocks to start with, when there are no conflicting definitions.  Defines don't use any space.  I don't understand the purpose of the per cam blocks, so it triggers my Chesterton's Fence alarm.